lib/src/commands/create/templates/template.dart

```diff
@@ -12,6 +12,7 @@ abstract class Template {
     required this.name,
     required this.bundle,
     required this.help,
+    required this.brick,
```

Comment:
From What I could see, the bricks are always generated from some bundle fields.  Shouldn't the constructors (on the `Template` subclasses) always receive the bundle only instead of receiving the brick also?

Reply:
Good call, should be resolved now 👍